### PR TITLE
Feature: Remove all png files

### PR DESCRIPTION
**Pull request by ashrafxcoder**
- I have removed the only file

### CODE REVIEW

**Comment by majidliaquat** [comment]
Okay Got it, there should be not any png file on this repository

**Comment by HuzaifDharejo** [comment]
Sure we are looking in to it :)

**Comment by Awais-Khan1829** [comment]
Woah.

**Review by majidliaquat — CHANGES_REQUESTED** [review]
Change the file as per the previous status

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# GitHub-Learning
\ No newline at end of file
+# GitHub-Learning
+
```
Comment (Awais-Khan1829) [inline README.md:2]: Okay

File: README.md [hunk 2]
```diff
@@ -1 +1,3 @@
...
+This project is just for learning GitHub
```
Comment (HuzaifDharejo) [inline README.md:3]: this looks good to me

**Review by Awais-Khan1829 — APPROVED** [review]
All Okay

**Pull request merged**